avocado/plugins/jobs.py

```diff
@@ -33,6 +34,28 @@ class Jobs(CLICmd):
     name = 'jobs'
     description = 'Manage Avocado jobs'
 
+    def _get_data_from_file(self, filename):
+        if not filename or not os.path.isfile(filename):
+            raise FileNotFoundError('File not found {}'.format(filename))
+
+        with open(filename, 'r') as fp:
+            return json.load(fp)
+
+    def _print_job_details(self, details):
+        for key, value in details.items():
+            print("{:15s}: {}".format(key, value))
+
+    def _print_job_tests(self, tests):
+        print("\nTests:")
```

Comment:
Maybe use https://avocado-framework.readthedocs.io/en/79.0/api/utils/avocado.utils.html#avocado.utils.astring.iter_tabular_output here?

Reply:
Hi @clebergnu very handy this method, didn't know about it. Thanks.

But, due to the nature of the current data (list of dict) I would need to prepare the data first before sending to this method. So I would be adding more lines here.

Would be nice to have a handy method like this to accept a list of dict. But I suggest to handle this on a different PR, if possible.